### PR TITLE
Fixing typos and grammer mistakes

### PR DESCRIPTION
**Pull request by kevinmmansour**
Fixing typos and grammer mistakes across all the platform.

### CODE REVIEW

File: app/views/flags/history.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h1>Flag History</h1>
 <p>
   Below is a list of all the flags you have cast and the outcome of each of them. Moderators handling your flags may also
-  have left you feedback.
+  have left you a feedback.
```
Comment (cellio) [inline app/views/flags/history.html.erb:4]: "Feedback" is a mass noun not a counting noun.  "A feedback" sounds odd to me.  "Feedback" or "a response" both work.

**Comment by cellio** [comment]
Aside from one place where I commented these wording changes look fine (and are clearer).  This isn't Python so I *assume* the couple of whitespace changes don't matter.

File: app/views/flags/history.html.erb (hunk 1 above)
Comment (kevinmmansour) [reply app/views/flags/history.html.erb]: OK. rolled back to "Feedback".

**Comment by kevinmmansour** [comment]
> 
> 
> Aside from one place where I commented these wording changes look fine (and are clearer). This isn't Python so I _assume_ the couple of whitespace changes don't matter.

Yes, changes to whitespace don't differ but sometimes they are kind of formatting but I don't see that I changed in whitespace in this pull request.

**Comment by cellio** [comment]
@ArtOfCode-  this looks fine to me.

**Pull request merged**